scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -38,6 +38,6 @@ core.storage.audit.gae_models_test
 core.storage.classifier.gae_models_test
 main_test
 core.storage.feedback.gae_models_test
-core.domain.user_domain_test
 core.storage.exploration.gae_models_test
+core.domain.user_domain_test
```

Comment:
Why did you modify this?

Reply:
Actually, this file didn't have complete coverage. But I had to modify the user_domain.py file, so I had to run tests for it and check coverage for the lines that I had modified. So, I deleted it, completed the coverage for my modifications and then added it back.

Reply:
My question is why you modified this line to line no 42. Earlier it was at line no 41.

Reply:
Oh. Does the order matter? I just wrote the tests for exploration.gae_models.py before the user_domain one. Should I change it back @SAEb-ai ?

Reply:
I don't think so it matters but why change something without any reason, right?

Reply:
Ya, will update it.